--- a/meson.build
+++ b/meson.build
@@ -204,16 +204,6 @@
         rename: 'swayimg.png',
         install_dir: get_option('datadir') / 'icons/hicolor/64x64/apps',
     )
-    install_data(
-        'extra/icon_128.png',
-        rename: 'swayimg.png',
-        install_dir: get_option('datadir') / 'icons/hicolor/128x128/apps',
-    )
-    install_data(
-        'extra/icon_256.png',
-        rename: 'swayimg.png',
-        install_dir: get_option('datadir') / 'icons/hicolor/256x256/apps',
-    )
 endif
 
 # zsh completion
